support/modules-artifact-gen/directory-artifact-gen

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
Another thing to pay attention when migrating from `bash` to `sh` is the if statements: `==` is bash syntax, `=` is sh syntax. I am getting the following error:
```
$ ./directory-artifact-gen -n hola -t hola --dest-dir /hola /home/lluis/Documents/ -- -T something
./directory-artifact-gen: 150: [: /hola: unexpected operator
Error: Conflicting flag '-T'. Already specified by the script.
```